Source/Blazorise/Models/ToastInstanceOptions.cs

```diff
@@ -50,6 +50,21 @@ public class ToastInstanceOptions
     /// Delay in milliseconds before hiding the toast.
     /// </summary>
     public double? AutohideDelay { get; set; }
+    
+    /// <summary>
+    /// Custom CSS class name to apply to the Toast
+    /// </summary>
+    public string ToastClass { get; set; }
```

Comment:
ToastInstance already represents a Toast component. So I would remove the Toast prefix from the name as it is redundant.